Source/WebCore/platform/SharedBuffer.cpp

```diff
@@ -121,6 +158,26 @@ Ref<SharedBuffer> FragmentedSharedBuffer::makeContiguous() const
     return SharedBuffer::create(WTFMove(combinedData));
 }
 
+auto FragmentedSharedBuffer::toIPCData() const -> IPCData
+{
+    size_t bufferSize = size();
+    if (!bufferSize)
+        return { };
```

Comment:
May be nicer to return an empty Vector instead? Otherwise, it constructs a variant with a SharedMemoryHandle that is nullopt, which should not be valid. However, sending an empty buffer should be OK I think.